openbb_terminal/config_terminal.py

```diff
@@ -96,11 +96,13 @@ def setup_logging_sub_app(sub_app: str):
     set_current_system(updated_system)
 
 
-def setup_config_terminal():
+def setup_config_terminal(app_name: str = "gst"):
```

Comment:
I feel that this can be misleading if the app_name is not actually "gst". Can we have something like `is_sdk: bool = False` instead?

Reply:
EZPZ good idea